suppressions.txt

```diff
@@ -1,2 +1,3 @@
 missingIncludeSystem
 useStlAlgorithm
+unusedFunction
```

Comment:
You're explicitly suppressed some functions in cpp(s)
And here you're excluding unused function for the whole project

So why not simply mark explicitly all those errors as suppressed ones, or exclude this error globally, without explicit excludes